Smarty/templates/Settings/LayoutBlockEntries.tpl

```diff
@@ -387,14 +388,80 @@
 									<td colspan="3" class="dvtCellInfo" align="center">
 										<input type="button" name="save" value=" &nbsp; {$APP.LBL_SAVE_BUTTON_LABEL} &nbsp; " class="crmButton small save" onclick="saveFieldInfo('{$value.fieldselect}','{$MODULE}','updateFieldProperties','{$value.typeofdata}');" />&nbsp;
 										{if $value.customfieldflag neq 0}
-											<input type="button" name="delete" value=" {$APP.LBL_DELETE_BUTTON_LABEL} " class="crmButton small delete" onclick="deleteCustomField('{$value.fieldselect}','{$MODULE}','{$value.columnname}','{$value.uitype}')" />
+											<input type="button" name="delete" value="{$value.label}" id="empid" class="crmButton small delete"  onclick="getData(this.value, '{$value.label}'); fnvshobj(this,'hiddenfield_{$value.label}'); " />
 										{/if}
 										<input type="button" name="cancel" value=" {$APP.LBL_CANCEL_BUTTON_LABEL} " class="crmButton small cancel" onclick="fninvsh('editfield_{$value.fieldselect}');" />
 									</td>
 								</tr>
 							</table>
 						</div>
 					</div>
+
+<script type="text/javascript" src="modules/Settings/script.js"></script>
```

Comment:
Please put this javascript code in the Settings.js file

https://github.com/tsolucio/corebos/blob/master/modules/Settings/Settings.js

that one is already included in the service worker